target/linux/ramips/dts/mt7621.dtsi

```diff
@@ -328,17 +346,30 @@
 		};
 	};
 
-	sdhci: sdhci@1e130000 {
+	sdhci: mmc@1e130000 {
 		status = "disabled";
 
-		compatible = "ralink,mt7620-sdhci";
+		compatible = "mediatek,mt7620-mmc";
```

Comment:
This new driver needs more tests. Last month, I tried to port it to MT7620 but it failed to probe sdcard.

Reply:
Couldn't this be left for another PR? I don't see why it's relevant for 6.1 support, unless the old drivers don't work on 6.1